src/service/user/userController.ts

```diff
@@ -1,46 +1,46 @@
-export {}
+export { }
 const express = require('express');
 const router = express.Router()
 const businessService = require('../business/businessService')
 const reviewService = require('../reviews/reviewService')
 const userService = require('../user/userService')
-const {authenticateToken} = require('../auth/user/userAuthorization.ts')
+const { authenticateToken } = require('../auth/user/userAuthorization.ts')
 
 //Get User Businesses
 router.get('/user/:userid/businesses', (req, res) => {
     try {
-        return res.status(200).json(businessService.getUserBusinesses(req.params.userId))
+        return res.status(200).json(businessService.getUserBusinesses(req.params.userid))
```

Comment:
Can you change this to user_id? I think I'm using underscores for the body params